### DISCUSSION TITLE
Welcome to QuantLib-Python-Docs Discussions!

### DISCUSSION THREAD

**Discussion by nhaga** [nhaga/QuantLib-Python-Docs#28] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by jackgillett101** [comment]
Just passed one year... happy birthday QuantLib-Python-Docs!

**Reply by nhaga** [reply]
QuantLib-Python-Docs is looking good... let's keep it up.

I have been experimenting with jupyter-book which would allow to build the docs as a collection of notebooks. It looks pretty good, would have the code output directly, and maybe easier to maintain.

I might set up another repository with it so anyone can try it out...
